5 oarhole lock annotation (#9)
Virtual-Viking-Longship/Ship-Annotations: Propulsion/Oarhole-Lock.md

**_oarhole-lock_** (English); _årehulslukke_ (Danish); _Riemenlochverschluss_ (German)

> _**ár** f., pl. árar_ (Old Norse) [citations: [prose](https://onp.ku.dk/onp/onp.php?o4167)/[poetry](https://lexiconpoeticum.org/m.php?p=lemma&i=4453)] --> change 

Reviewer: ```suggestion
_unattested_ (Old Norse)
```

>   A small wooden tool used to to close the oarholes in the hulls of viking ships when the oars weren't in use, in order to prevent water from flooding the hull.     

Reviewer: ```suggestion
  A small wooden device used to to close the oarholes when not in use to prevent the ship taking on water.     
```
